boards/st/nucleo_u083rc/doc/index.rst

```diff
@@ -176,10 +197,10 @@ Default Zephyr Peripheral Mapping:
 - I2C1_SDA : PB9
 - LPUART_1_TX : PG7
 - LPUART_1_RX : PG8
-- SPI1_NSS : PA4
-- SPI1_SCK : PA5
-- SPI1_MISO : PA6
-- SPI1_MOSI : PA7
+- SPI1_NSS : PA15
+- SPI1_SCK : PA1
+- SPI1_MISO : PA11
+- SPI1_MOSI : PA12
```

Comment:
@ExaltZephyr Can you comment on this ?

Reply:
@erwango 
I noticed that PA6 and PA7 conflict with the pins assigned to I2C2 in this file,so I made adjustments accordingly.
We now have two options: either retain these pin assignments and disable I2C2, or keep I2C2 enabled and reassign the pins ?

Reply:
If possible to have Arduino SPI, Arduino I2C and USB all together, please select this option.
Otherwise please comment I2C and we'll enable it in tests

Reply:
I2C2 is not present on the Arduino connector, only I2C1 is available. 
Therefore, I will proceed with the second option and disable I2C2.